lib/chat/db/write_queue.ex

```diff
@@ -95,24 +99,27 @@ defmodule Chat.Db.WriteQueue do
     |> noreply()
   end
 
-  def handle_cast({:mirror, sink}, state) do
+  def handle_cast({:mirrors, sink}, state) do
     state
-    |> q_state(mirror: sink)
+    |> q_state(mirrors: sink)
```

Comment:
rewriting previous mirror?

Reply:
What do you mean? The only difference is now the mirror is a list of writers.

Reply:
Fixing this.

Reply:
Actually, it seems to work as expected. For InternalDB and BackupDB, mirrors get set to `nil`, and for MainDB it gets set to `[Chat.Db.InternalDb.Writer, Chat.Db.BackupDb.Writer]`.

Reply:
It is odd that it works
May we have a test case for this?

Reply:
Added tests for main and backup sync in platform's PR.